packages/mroonga-package-task.rb

```diff
@@ -235,23 +235,42 @@ def split_mysql_package
     [names.join("-"), series]
   end
 
-  def detect_mysql_community_rpm_version
-    series = split_mysql_package[1]
-    # Use the URL without a trailing slash because the CDN caches both “…/SRPMS”
-    # and "…/SRPMS/", but the cache for the "/SRPMS/" path hasn't been expired
-    # yet. Using the no‑slash URL ensures we get the latest listing.
+  def latest_target_version(series, trailing_slash, minimal_or_not = nil)
```

Comment:
```suggestion
  def latest_target_version(series, trailing_slash: false, minimal: false)
```

Call method: `latest_target_version(series, trailing_slash: true, minimal: true)` 

It seems better to use boolean in this way.